docs/user/index.md

```diff
@@ -58,18 +58,7 @@ action objects if needed. Each of those actions will be executed when the rule i
 only for getting several actions executed.
 
 Last but no least, the `text` field contains the valid EPL statement to be send to the Esper-based core rule engine. The
-value of this field must follow the EPL syntax, that is fully documented in the
-[Esper website](http://esper.espertech.com/release-6.1.0/esper-reference/html/index.html), in
-[here](http://esper.espertech.com/release-6.1.0/esper-reference/html/epl_clauses.html).
-
-Perseo should work with any of the EPL clauses. However, you should take into consideration the following guidelines:
-
--   The name of the stream from which events will come is `iotEvent`.
--   A `type=` condition should be included to avoid mixing different kinds of entities.
--   Entity's attributes must be cast to `float` in case of being numeric, to `String` otherwise.
--   All the attributes of the context broker notification are available as dynamic properties of the event object `ev`.
--   A question mark `?` is _necessary_ for EPL to refer to ‘dynamic’ values (duck typing), such as `ev.id?`.
--   Metadata is also available as explained in the [metadata and object values](#metadata-and-object-values) section.
+value of this field must follow the EPL syntax.
```

Comment:
I understand that the block above has been removed because it is redundant with others parts in the documentaiton where EPL is discussed in more detail. However, I think it would be useful to convert "EPL syntax" to the corresponding section (#epl-text in plain_rules.md?)

Reply:
Since this is a quick overview of the 3 elements composing the rule, the link to the documentation (full plain rules file) is at the bottom of the section. Otherwise, specific links could be provided on each section.